### DISCUSSION TITLE
nginx + authelia: rewrite after authelia ?

### DISCUSSION THREAD

**Discussion by leisefuxX** [authelia/authelia#3942] [Q&A] [answered]
before i used authelia for my app, i had a rewrite rule in nginx for my site, so url was cleaner.
i rewrote all from "domain.tld/mysuperapp" to "domain.tld".

the rule was: `location = /{return 301 $scheme://$http_host/myapp/;}
`

with authelia, i can't rewrite my urls and have to use the location, like "myapp.domain.tld/mysuperapp"

is there a way to add the same rewrite-behaviour with authelia? so basically when someone requests myapp.domain.tld he is forwarded to myapp.domain.tld/mysuperapp

?

**Comment by james-d-elliott** [comment, marked as answer]
Authelia doesn't specifically affect this, it has no knowledge of the nginx config. This is entirely an interaction with nginx rules unless I'm missing something. What do you mean you can't rewrite the URLs?
